xml/System.Text/SpanRuneEnumerator.xml

```diff
@@ -83,9 +89,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Advances the enumerator to the next <see cref="T:System.Text.Rune" /> of the span.</summary>
+        <returns>`true` if the enumerator successfully advanced to the next item; `false` if the end of the span has been reached.</returns>
```

Comment:
The backtick character is to escape in remarks (markdown). Inside normal tags, we use `<see langword="" />` for reserved keywords:

```suggestion
        <returns><see langword="true" /> if the enumerator successfully advanced to the next item; <see langword="false" /> if the end of the span has been reached.</returns>
```

Reply:
This is copied directly from `Span<T>.Enumerator.MoveNext`. Should it be fixed there as well? https://github.com/dotnet/dotnet-api-docs/blob/f2edeb89431d53aac9ae21fafaeb71a0c9e29de7/xml/System/Span%601%2BEnumerator.xml#L118

Reply:
Yes, I'll fix that @GrabYourPitchforks. Good catch!

Reply:
This is fixed now in master:
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Span%601%2BEnumerator.xml#L118